cmd/common-main.go

```diff
@@ -166,6 +166,10 @@ func minioConfigToConsoleFeatures() {
 				os.Setenv("CONSOLE_PROMETHEUS_EXTRA_LABELS", value)
 			}
 		}
+		// Support Prometheus Auth Token
+		if value := env.Get("MINIO_PROMETHEUS_AUTH_TOKEN", ""); value != "" {
```

Comment:
We need to define this at internal/config/constants.go

Reply:
Sure, Let me include the other vars as well

Reply:
Fixed